doc/contributing/writing-tests.md

```diff
@@ -141,6 +141,26 @@ request. Interesting things to notice:
 
 ## General recommendations
 
+### Usage of `node:test`
+
+It is optional to use `node:test` in tests outside of testing the `node:test`
```

Comment:
It seems important to document things like https://github.com/nodejs/node/pull/52177 or otherwise we would see more flakes coming up once people start to spawn hundreds of child processes and overloading the machine using `spawnPromisified` + `it()` or `test()`....

Reply:
Agreed. Using the concurrency option is fine though unless you are specifically planning to spawn child processes. But that applies to things like `Promise.all()` as well.

Reply:
Can you recommend changes to the text, please?